package/skelcd-control-SLES.spec

```diff
@@ -70,6 +70,9 @@ Requires:       yast2-users
 Requires:       yast2-x11
 # Ruby debugger in the inst-sys (FATE#318421)
 Requires:       rubygem(%{rb_default_ruby_abi}:byebug)
+# Install and enable xrdp by default (FATE#320363)
+Requires:       yast2-rdp
```

Comment:
As the comment on line 30 says, these are the packages that are  needed in the inst-sys. Is that really the case for yast2-rdp? From reading the feature I think it is not.